<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TruckerChecksDialog</class>
 <widget class="QDialog" name="TruckerChecksDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>546</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Print Trucker Checks</string>
  </property>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;p align=&quot;center&quot;&gt;Seller Truckers&lt;/p&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>510</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Selected</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>520</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;p align=&quot;center&quot;&gt;Buyer Truckers&lt;/p&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3ListBox" name="buyerTruckerList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>520</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans Mono</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>Q3ListBox::Multi</enum>
   </property>
  </widget>
  <widget class="Q3ListBox" name="sellerTruckerList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>520</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans Mono</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>Q3ListBox::Multi</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="reportCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print A Report</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/truckingrecorddao.h</include>
  <include location="local">types/truckingrecord.h</include>
  <include location="local">dao/invoicedao.h</include>
  <include location="local">dao/setupdao.h</include>
  <include location="local">reports/textcheck.h</include>
  <include location="local">dao/checkdao.h</include>
  <include location="local">lineprinter.h</include>
  <include location="local">reports/truckercheckreport.h</include>
  <include location="local">dao/invoicedao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
